Functions/Administration/xASL_adm_SortDicomToFolders.m

```diff
@@ -47,50 +47,53 @@ function xASL_adm_SortDicomToFolders(pathDICOM, nDirLayers, bUseDCMTK, bVerbose)
 	end
 else
 	% Gather a list of all files in the directory
-	Flist   = xASL_adm_GetFileList(pathDICOM,'^.*.(?!(xlsx|ini|json))$','FPListRec',[0 Inf]);
+	Flist   = xASL_adm_GetFileList(pathDICOM,'.*','FPListRec',[0 Inf]);
 	for iL=1:length(Flist)
-		% Track progress if verbosity is on
-		if bVerbose; xASL_TrackProgress(iL, length(Flist)); end
+		[~, ~, Fext] = xASL_fileparts(Flist{iL});
+		if isempty(regexp(Fext, '(xlsx|ini|json|DS_Store)'))
+			% Track progress if verbosity is on
+			if bVerbose; xASL_TrackProgress(iL, length(Flist)); end
```

Comment:
The rest seems all the same. To me, this is a useless fix. Unless you explain me why this was needed ;)
The only thing I see is the index file DS_Store that macOS adds, but that could easily be added on the top list of the files that should not be listed.